lib/widgets/theme_switch.dart

```diff
@@ -16,7 +13,9 @@ class ChangeThemeTile extends StatelessWidget {
     final themeProvider = Provider.of<AppTheme>(context);
     return ListTile(
       key: const Key('ThemeSwitch'),
-      contentPadding: EdgeInsets.zero,
+      contentPadding: EdgeInsets.symmetric(
+        horizontal: SizeConfig.blockSizeHorizontal! * 1.5,
```

Comment:
seems like you are using it in multiple places when not make it a constraint variable